nbbuild/external/codesnippet-doclet-0.30-license.txt

```diff
@@ -0,0 +1,684 @@
+Name: Codesnippet Javadoc Doclet
+Version: 0.30
+License: GPL-3
+Source: https://github.com/jtulach/codesnippet4javadoc/
+Description: Javadoc doclet to include real code snippets in the documentation
+Origin: apidesign.org
+Type: compile-time
```

Comment:
Oh, so we are supposed to accept it since it's only compile time?

Reply:
Right. It is compile time and no Apache code is linked to that library. No Apache code is derived work of that library. Also, in the readme of the codesnippet4javadoc I explicitly allow anyone to use the tool to generate Javadoc for any code of any license.